bot/tests/test_notifier.py

```diff
@@ -50,5 +50,5 @@ def test_notification(mock_secrets, mock_taskcluster, mock_phabricator, fake_hg_
     mail = notify_email(revision, stack, changesets_coverage)
     assert (
         mail
-        == "* [Commit [(b'M', b'file')]Differential Revision: https://phabricator.services.mozilla.com/D2](https://phabricator.services.mozilla.com/D2): 0 covered out of 6 added.\n"  # noqa
+        == "* [Commit [(b'M', b'file')]Differential Revision: https://phabricator.services.mozilla.com/D2](https://phabricator.services.mozilla.com/D2): 0 covered out of 6 added.\n"
```

Comment:
Please revert this change and the change in bot/tests/test_taskcluster.py, as they are unrelated.